build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -48,6 +59,15 @@ def substituteMultiline(multilineLogging, stacktraceLanguages, new_contents)
     return new_contents
 end
 
+def substituteStorageTotalLimitSize(new_contents)
+   if is_high_log_scale_mode?
+      new_contents = new_contents.gsub("${STORAGE_TOTAL_LIMIT_SIZE_MB}", "storage.total_limit_size        1000M")
```

Comment:
do we want to change the new added config to this format in fluent-bit conf? #${STORAGE_TOTAL_LIMIT_SIZE_MB} storage.total_limit_size        1000M     and then here: new_contents = new_contents.gsub("#${STORAGE_TOTAL_LIMIT_SIZE_MB}", "")